src/js/providers/html5.js

```diff
@@ -125,6 +128,12 @@ function VideoProvider(_playerId, _playerConfig) {
                 offset: offset
             });
             _setPositionBeforeSeek(offset);
```

Comment:
At this point where "seeking" is fired, `currentTime` has already changed. This line should be removed.

Reply:
@robwalch this is used when seeking in fullscreen iOS. On iOS fullscreen you can only scrub so you get multiple seeking events as you scrub. if we don't update the position after each seek event, the position in the callback is never updated.

Reply:
so you end up having onSeek {position 0 - offset 1} , onSeek {position 0 - offset 2} , onSeek {position 0 - offset 3}
instead of onSeek {position 0 - offset 1} , onSeek {position 1 - offset 2} , onSeek {position 2 - offset 3}

Reply:
Shouldn't we update it on "seeked" then?

Reply:
@robwalch seeked is only fired at the end of scrubbing, so when removing `_setPositionBeforeSeek(offset);` we're left with
<img width="541" alt="screen shot 2017-10-31 at 12 27 25 pm" src="https://user-images.githubusercontent.com/12138319/32235914-1fb3001c-be37-11e7-9f2a-b427871d3938.png">

it seemed confusing to me but if you think it's fine i don't mind removing the line